README.md

```diff
@@ -502,7 +502,7 @@ There's several steps to getting this working:
 
 **1. Gunicorn deployment**:
 
-The `prometheus_multiproc_dir` environment variable must be set to a directory
+The `PROMETHEUS_MULTIPROC_DIR` environment variable must be set to a directory
```

Comment:
Let's keep the README value lowercase for at least a release or two so people will not be confused if using an older version.

Reply:
Would be good to start a changelog.md to keep tracks of changes between versions.